src/iroha_client_wrap.rs

```diff
@@ -6,27 +6,31 @@ use color_eyre::{
     Result,
 };
 use iroha_client::{
-    client::{Client as IrohaClient, ClientQueryError, ClientQueryOutput, ResponseHandler},
+    client::{Client as IrohaClient, ClientQueryError, ClientQueryRequest, ResponseHandler},
     http::Response as RespIroha,
 };
+use iroha_core::tx::Executable;
 use iroha_data_model::prelude::Sorting;
 use iroha_data_model::{
     metadata::UnlimitedMetadata,
     predicate::PredicateBox,
-    prelude::{Instruction, Pagination, Query, QueryBox, Value},
+    prelude::{InstructionBox, Pagination, Query, QueryBox, Value},
 };
 use iroha_telemetry::metrics::Status;
 
 use request_builder::ActixReqBuilder;
 
 mod request_builder {
+
```

Comment:
I guess you haven't run `cargo fmt`. Please do.

Reply:
Also, don't forget about using `cargo clippy` as well. It is a great automatic tool to help you learn Rust and to write idiomatic Rust.

Reply:
@horizenight, what about this?